imap_processing/cdf/cdf_attribute_manager.py

```diff
@@ -236,10 +238,22 @@ def get_variable_attributes(self, variable_name: str) -> dict:
         Returns
         -------
         dict
-            I have no idea todo check.
+            Information containing specific variable attributes
+            associated with "variable_name".
         """
-        # TODO: Create a variable attribute schema file, validate here
-        if variable_name in self.variable_attributes:
-            return self.variable_attributes[variable_name]
-        # TODO: throw an error?
-        return {}
+        output = dict()
+        for attr_name in self.variable_attribute_schema.keys():
```

Comment:
I don't think that this attribute schema validation is working correctly. The reason that I say this is on Hi, I define a variable attribute `dtype` in all of my variable attribute dictionaries that I use when instantiating the xarray.DataArray objects that store the Hi data. Since `dtype` is not defined in the variable attribute schema, I would expect this change to break several Hi processing tests.

I would suggest adding tests that verify the output dictionary looks as expected when the condition `variable_name in self._variable_attributes[variable_name]` is True.